Fix config changes (#27)
wso2/docs-ob-toolkit-cds: en/docs/install-and-setup/configuring-identity-server-for-ob.md

    secure="false"
    http_method="GET,DELETE"
    ```

10. Add and configure the following tags:

    - `enable` : To enable metadata caching, set the value to `true`.
    - `update_period` : Configure the fetch time for periodical cache update in minutes.
    - `data_recipient_discovery_url` : Configure the endpoint provided by ACCC for retrieving the statuses of Data Recipients.

>     - `dcr_internal_url` : Configure `<IS_HOST>` placeholder with the hostname of the Identity Server.

Reviewer: ```suggestion
    - `dcr_internal_url` : Configure the `<IS_HOST>` placeholder with the hostname of the Identity Server.
```

>     - `apim_application_search_url` : Configure `<APIM_HOST>` placeholder with the hostname of the API Manager. This endpoint will

Reviewer: ```suggestion
    - `apim_application_search_url` : Configure the `<APIM_HOST>` placeholder with the hostname of the API Manager. This endpoint will
```

>     - `apim_application_search_url` : Configure `<APIM_HOST>` placeholder with the hostname of the API Manager. This `application search admin` endpoint will

Reviewer: ```suggestion
    - `apim_application_search_url` : Configure the `<APIM_HOST>` placeholder with the hostname of the API Manager. This `application search admin` endpoint will
```

>     - `apim_application_search_url` : Configure `<APIM_HOST>` placeholder with the hostname of the API Manager. This `application search admin` endpoint will

Reviewer: ```suggestion
    - `apim_application_search_url` : Configure `<APIM_HOST>` placeholder with the hostname of the API Manager. This endpoint will
```

>       returns the list of client applications. For more details, see

Reviewer: ```suggestion
      return a list of client applications. For more details, see
```
